pyproject.toml

```diff
@@ -138,9 +140,17 @@ lint.allowed-confusables = [
 lint.ignore = [
     "COM812", # missing-trailing-comma (ruff-format recommended)
     "D203", # one-blank-line-before-class
+    "D205",
     "D212", # blank-line-before-class
+    "D401",
+    "EM101",
+    "EM102",
     "ISC001", # single-line-implicit-string-concatenation (ruff-format recommended)
+    "PLR0912",
+    "PLR0913",
     "RUF003", # ambiguous-unicode-character-comment
+    "SIM108",
+    "TRY003",
```

Comment:
I would prefer keeping these

Reply:
I don't know. Is a `a if cond else b` ever more readable than separate lines?

TRY003 I think is actually harmful. If I see a stack trace, I want to see that the error message actually came from the line that raised, and wasn't forwarded from a caught exception or similar.